README.md

```diff
@@ -1,43 +1,91 @@
+- SolidOS Databrowser Frontend - a frontend for Solid Servers like on <solidcommunity.net>
+- SolidOS Data-Kitchen - a stand-alone desktop app: <https://github.com/solid/data-kitchen<>.
 
-- When used with a Solid pod, because that is on the web, the data browser provides the **public view** -- the interactive interface -- that the user has with the rest of the world.  Like when everyone had their own home page on the web, they have that power again to express themselves and their affiliation and their products, and to court interaction, such as collaborative work with others, or commerce.  The way this public home page appears to others is very customizable, so the user, individual or business can be proud of it.
+And mashlib is also used standalone as the SolidOS Databrowser Webapp and can be tried out at: <https://solid.github.io/mashlib/dist/browse.html>.
 
-- The data browser should be modular, **dynamically** loading new code modules in real time as a function of a user's preferences for handling different types of data with different new data browser applets, be it finance, fitness, or fishing.
+But mashlib is also used as a library by adding `mashlib.js` (of minified version) directly in your applications. For example:
 
-- A module providing new functionality in a new domain should be able to appear as a module in the data browser or as a stand-alone app, or both.
+`<script src="https://solidcommunity.net/mashlib.js"></script>`.
 
-- The modularity of the system should allow you set yourself up with any set of apps, or indeed the user should be able to configure the data browser to replace itself with the user's own choice of alternative data browser.  All data browsers should allow the user to change this selection.
+## Previous versions of this documentation
 
-- The data browser should allow people to create, bit by bit, a web of social linked data of their work and their play, and their lives.
+Check out [SolidOS Pod](https://solidos.solidcommunity.net/Team/docs/solidos.html) for an earlier version of this documentation.
 
+# Documentation
 
-## Typical uses
+## Different implemnetation
+
+### SolidOS Databrowser Webapp
+
+The `static/browse.html` page is compiled one to one into the `dist` (output) folder of mashlib and makes mashlib available stand-alone as the SolidOS Databrowser Webapp.
+
+You can see and try out a SolidOS Databrowser Webapp deployment under this link: <https://solid.github.io/mashlib/dist/browse.html>
+
+`browse.html`serves as a perfect example for: Solid WebID authentication and how to make use of mashlib function and variables. 
+
+### SolidOS Databrowser Frontend
+
+The `src/databrowser.html`page is compiled into the SolidOS Databrowser Frontend which is displayed for each WebID on [solidcommunity.net](https://solidcommunity.net/). This is the case because the [solidcommunity.net](https://solidcommunity.net/) Solid Server is configured with SolidOS as its frontend.
```

Comment:
```suggestion
The `src/databrowser.html`page is compiled into the SolidOS Databrowser Frontend which is displayed for each WebID on [solidcommunity.net](https://solidcommunity.net/). This is the case because the [solidcommunity.net](https://solidcommunity.net/) Solid Server is configured with SolidOS as its front-end.
```